### DISCUSSION TITLE
How to compile language packs?

### DISCUSSION THREAD

**Discussion by Century1026** [inventree/InvenTree#7266] [Q&A]
I can't wait to put the project into use, I decided to start translation right now, but how can I compile the django.po file?

**Comment by SchrodingersGat** [comment]
You should not have to do this yourself, compiled translations are periodically merged into the master branch.

If you want to contribute to the InvenTree translation effort, you can [refer to our website](https://inventree.org/contribute.html#translate)

**Reply by Century1026** [reply]
Yes sir! Actually, I used to contribute to the InvenTree translation and I noticed that submitted translation needs further confirmation. So, I'm wondering is there a routine for them to be merged?

**Reply by SchrodingersGat** [reply]
I believe that even "pending" translations are merged back into the code base
